examples/computer_vision_techniques/off_limb_enhance.py

```diff
@@ -49,18 +49,19 @@
 # Let's plot the results using LaTeX for all the text.
 
 fontsize = 14
-plt.plot(rsun_array, y, label='data')
+fig, ax = plt.subplots()
```

Comment:
```suggestion
fig = plt.figure()
ax = fig.add_subplot()
```

My initial reaction was to make this replacement everywhere, but I don't think the added verbosity is worth the relatively minor gain in consistency.

Reply:
We value consistency!